and-bible/app/src/main/java/net/bible/service/device/speak/TextToSpeechServiceManager.java

```diff
@@ -396,9 +490,15 @@ public void onInit ( int status){
 			if (mTts != null && status == TextToSpeech.SUCCESS) {
 				Log.d(TAG, "Tts initialisation succeeded");
 
+				// Add earcons
+				mTts.addEarcon(EARCON_PRE_FOOTNOTE, BibleApplication.getApplication().getPackageName(), R.raw.short_pling); // TODO: change
+				mTts.addEarcon(EARCON_POST_FOOTNOTE, BibleApplication.getApplication().getPackageName(), R.raw.short_pling_reverse);
+				mTts.addEarcon(EARCON_PRE_TITLE, BibleApplication.getApplication().getPackageName(), R.raw.pageflip);
```

Comment:
Do you have any ideas for appropriate sounds? I will try to find nicer sounds from my synthesizer keyboard etc. Earcon sounds could perhaps be also made configurable, but I'm not sure how useful that would be.

Reply:
I think that originally they were reversed, which would seem reasonable.  If you think the page turning sound is too intrusive then could you just shorten it a bit?

Reply:
I actually like the page turning sound -- it is somehow more natural when listening to speech.
On the other hand, I don't like "pling" sounds -- they are more intrusive. When listening to bible, title sound (now pageflip) are frequently played, but other sounds less frequently. That's why I chose pageflip for title sound. 

But I agree, it can be a little confusing to user as it kind of gives wrong cue about what's happening (no, page is not flipping actually, but there's a new title)...

- [ ] I'll think about another sound for titles.